projects/canopy/src/lib/card/docs/card/guide.stories.mdx

```diff
@@ -103,6 +103,98 @@ The main body of the card. It accepts the projection of the following components
 - `LgCardContent`
 - `LgCardFooter`
```

Comment:
I think you should add `LgCardHeroImageComponent` here too, no?

Reply:
Done. :)